src/migrations/hotbar-store/5.0.0-alpha.0.ts

```diff
@@ -20,34 +20,24 @@
  */
 
 // Cleans up a store that had the state related data stored
-import type { Hotbar } from "../../common/hotbar-store";
-import { ClusterStore } from "../../common/cluster-store";
+import { Hotbar, HotbarStore } from "../../common/hotbar-store";
 import * as uuid from "uuid";
 import type { MigrationDeclaration } from "../helpers";
+import { catalogEntity } from "../../main/catalog-sources/general";
 
 export default {
   version: "5.0.0-alpha.0",
   run(store) {
-    const hotbars: Hotbar[] = [];
+    const hotbar: Hotbar = {
+      id: uuid.v4(),
+      name: "default",
+      items: HotbarStore.getInitialItems(),
+    };
 
-    ClusterStore.getInstance().clustersList.forEach((cluster: any) => {
-      const name = cluster.workspace;
+    const { metadata: { uid, name, source } } = catalogEntity;
 
-      if (!name) return;
+    hotbar.items[0] = { entity: { uid, name, source } };
 
-      let hotbar = hotbars.find((h) => h.name === name);
-
-      if (!hotbar) {
-        hotbar = { id: uuid.v4(), name, items: [] };
```

Comment:
Furthermore, I don't know when this code was added but this is wrong since `name` is misleading: `cluster.workspace` is not the workspace name, but in fact its a workspace ID.